commit c1ef9f5389e5debb132b7e2ab40f178ed413a978
Author: Lukas Tribus <lukas@ltri.eu>
Date:   Wed Oct 17 01:40:11 2018 +0200

    DOC: fix reference to map files in MAINTAINERS
    
    s/maps/map
    
    (cherry picked from commit b75e828b298c958beb10c830a1ccb3df0840c30c)
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/MAINTAINERS b/MAINTAINERS
index 3a9e435a..df7cc336 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -24,7 +24,7 @@ Files: src/hlua.c, include/*/hlua.h
 
 Maps and pattern matching
 Maintainer: Thierry Fournier <tfournier@arpalert.org>
-Files: src/maps.c, src/pattern.c, include/*/maps.h, include/*/pattern.h
+Files: src/map.c, src/pattern.c, include/*/map.h, include/*/pattern.h
 
 DNS
 Maintainer: Baptiste Assmann <bedis9@gmail.com>
